stdparams.go

```diff
@@ -0,0 +1,38 @@
+// +build go1.7
+
+package httprouter
+
+import (
+	"context"
+	"net/http"
+)
+
+type paramsKey struct{}
```

Comment:
@fgrosse their is a actual difference. There are two benefits to using the technique.
  1. The key can never be set outside the package, useful for overlapping key names.
  2. In the Context.Value an `interface{}` type is used as a key. An interface has two "values" in it, first the defined and then a pointer to the data (with an actual type), see http://research.swtch.com/interfaces. An as `struct{}` always has the same address (see https://play.golang.org/p/U-5LyKlCc0) it won't use extra memory for the actual data.